src/model/profile.rs

```diff
@@ -233,6 +261,140 @@ impl Profile {
+        let mut equipped_gear = Vec::with_capacity(14);
+        for item in attr_block.find(Class("js__db_tooltip")) {
+            // childless div = empty slot
+            if item.first_child().is_none() {
+                equipped_gear.push(None);
+                continue;
+            }
+
+            let mut slot = Slot::default();
+            slot.name = {
+                match item.find(Class("db-tooltip__item__name")).next() {
+                    Some(node) => Some(node.text()),
+                    None => None,
+                }
+            };
+            slot.glamour_name = {
+                // using the `view item details` button on hover, and fetching the parent <p>'s text
+                match item.find(Class("db-tooltip__item__mirage__btn")).next() {
+                    Some(node) => Some(
+                        node.parent()
+                        .unwrap()
+                        .text()
+                    ),
+                    None => None,
+                }
+            };
+            slot.ilvl = {
+                match item.find(Class("db-tooltip__item__level")).next() {
+                    Some(node) => Some(
+                        // comes in format `Item Level 630`
+                        node.text()
+                        .replace("Item Level ", "")
+                        .parse::<u32>()
+                        .unwrap_or(0)
+                    ),
+                    None => None,
+                }
+            };
+            equipped_gear.push(Some(slot));
+        };
+
+        Ok(
+            (PlayerInfo {
+                class,
+                level,
+            },
+            EquippedGear {
+                mainhand:       equipped_gear[0].clone(),
+                head:           equipped_gear[1].clone(),
+                body:           equipped_gear[2].clone(),
+                hands:          equipped_gear[3].clone(),
+                legs:           equipped_gear[4].clone(),
+                feet:           equipped_gear[5].clone(),
+                facewear:       equipped_gear[6].clone(),
+                offhand:        equipped_gear[7].clone(),
+                earrings:       equipped_gear[8].clone(),
+                necklace:       equipped_gear[9].clone(),
+                bracelets:      equipped_gear[10].clone(),
+                ring_left:      equipped_gear[11].clone(),
+                ring_right:     equipped_gear[12].clone(),
+                soul_crystal:   equipped_gear[13].clone(),
+            }
+        ))
+    }
+
+    fn parse_fieldops(doc: &Document) -> Result<FieldOps, Error> {
+        let attr_block = ensure_node!(doc, Class("character__content"));
+
+        // if not unlocked, the corresponding div is absent. otherwise its all there
+        let bozja: Option<u32> = {
+            match doc.find(Class("xiv-lds-resistance-level")).next() {
+                Some(node) => Some(
+                    node
+                    .parent().unwrap()
```

Comment:
not the neatest method, I'm aware,
but the structure holds true as long as `xiv-lds` is found